packages/gap-web-ui/src/components/summary-list/SummaryList.tsx

```diff
@@ -15,59 +17,77 @@ export interface Row {
   action?: string | JSX.Element;
 }
 
+const STYLE_MAP = {
+  displayRegularKeyFont: 'gap-summary-list--key-weight-regular',
+  hasWiderKeyColumn: 'key-width-45percent-sm',
+  equalColumns: 'equal-columns',
+  useBoldFont: 'fw-bold',
+  actionTextLeft: 'text-left',
+};
```

Comment:
I miss tailwind

Reply:
We would need to have this same pattern / style map with tailwind tbh, there's too many ui scenarios for this poor table component.